src/python/grpcio/grpc/beta/_client_adaptations.py

```diff
@@ -159,9 +159,11 @@ def time_remaining(self):
     return self._call.time_remaining()
 
   def add_abortion_callback(self, abortion_callback):
-    registered = self._call.add_callback(
-        lambda: abortion_callback(_abortion(self._call)))
-    return None if registered else _abortion(self._call)
+    def done_callback(unused_arg):
+      if self.code() is not grpc.StatusCode.OK:
```

Comment:
Never mix explicitly-return-a-value and implicitly-return-None-by-falling-off-the-end-of-the-method. Why does this function return a value anyway?